### PR TITLE
Updating SSDA Data release date

### PR DESCRIPTION
**Pull request by Eb-Zeero**
This change is [Reviewable](https://reviewable.io/reviews/saltastroops/salt-api/296)

### CODE REVIEW

File: saltapi/service/proposal_service.py [hunk 1]
```diff
@@ -374,6 +375,18 @@ def update_proprietary_period(
...
+        body = {
+            "query": "mutation($proposalCode:String!,$institution:Institution!,$apiKey:String!,$releaseDate:String!){updateReleaseDates(proposalCode:$proposalCode,institution:$institution,apiKey:$apiKey,metadataReleaseDate:$releaseDate,dataReleaseDate:$releaseDate){status}}",
+            "variables": {
+                "proposalCode": proposal_code,
+                "institution": "SALT",
+                "releaseDate": str(release_date),
+                "apiKey": get_settings().ssda_api_key
+            }
+        }
+        requests.post(get_settings().ssda_api_url, json=body)
```
Comment (hettlage) [inline saltapi/service/proposal_service.py:388]: You aren't handling errors.

File: saltapi/settings.py [hunk 2]
```diff
@@ -110,6 +110,12 @@ class Settings(BaseSettings):
     # URL for getting the TCS ICD file
     tcs_icd_url: str
 
+    # SSDA API key for updating proprietary period
```
Comment (hettlage) [inline saltapi/settings.py:113]: period > periods

File: saltapi/settings.py [hunk 3]
```diff
@@ -110,6 +110,12 @@ class Settings(BaseSettings):
...
+    ssda_api_key: str
+
+    # URL for the SAAO SALT Data Archive api
```
Comment (hettlage) [inline saltapi/settings.py:116]: api > API

**Review by hettlage — CHANGES_REQUESTED** [review]

File: saltapi/service/proposal_service.py (hunk 1 above)
Comment (Eb-Zeero) [reply saltapi/service/proposal_service.py:388]: i need what ever error the SSDA raises wich will course 500 internal server error whic exceptions_handles take care of

Comment (hettlage) [reply saltapi/service/proposal_service.py:388]: Ah, sorry, I forgot about that.

**Review by hettlage — APPROVED** [review]

**Pull request merged**